app/domain/authentication/authn_jwt/restriction_validation/create_constraints_from_policy.rb

```diff
@@ -0,0 +1,104 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    module RestrictionValidation
+      CreateConstrainsFromPolicy = CommandClass.new(
```

Comment:
Consider change to `CreateConstrainsFromConfiguration` the term FromPolicy looks a bit strange

Reply:
or just `CreateConstrains`

Reply:
I will call it CreateConstrainsFromPolicy